pimd/pim_nht.c

```diff
@@ -523,6 +554,7 @@ static int pim_ecmp_nexthop_search(struct pim_instance *pim,
 
 	// Current Nexthop is VALID, check to stay on the current path.
 	if (nexthop->interface && nexthop->interface->info &&
+	    ((struct pim_interface *)nexthop->interface->info)->options &&
```

Comment:
`->options` is no longer used after #10960

Reply:
Done